cmd/process-agent/collector.go

```diff
@@ -125,6 +128,11 @@ func NewCollectorWithChecks(cfg *config.AgentConfig, checks []checks.Check, runR
 	podResults := api.NewWeightedQueue(queueSize, int64(cfg.Orchestrator.PodQueueBytes))
 	log.Debugf("Creating pod check queue with max_size=%d and max_weight=%d", podResults.MaxSize(), podResults.MaxWeight())
 
+	dropConnectionsCheck := ddconfig.Datadog.GetBool("process_config.drop_connections_check")
+	if dropConnectionsCheck {
```

Comment:
Once we settle on the name, could we also add it to the `process-agent status` output?